ansible/collections/bento/common/roles/opensearch_dataloading/vars/main.yml

```diff
@@ -0,0 +1,14 @@
+# vars file for opensearch data loading
+#clickhouse_path: 
```

Comment:
since this script is opensearch specific the clickhouse references should be removed